### DISCUSSION TITLE
Communication interface, PID control window, setting preheat value

### DISCUSSION THREAD

**Discussion by romanplains** [artisan-roaster-scope/artisan#1146] [Q&A]
I am a complete novice so excuse me if I ask for something that should be obvious.

I just bought Hottop B+, downloaded Artisan, reviewed the documentation, YouTube presentations, and then proceeded to setup.
I run Artisan on MacBook Pro 2017 OS 13.2.1 .  I found several features offered by Artisan fantastic, particularly the ability to have a template of a roast, the designer, and PID.  At that moment, I thought, the remaining task is empirical - keeping some variables constant while varying others, gaining the insight into their relative importance and so on.  However, I run into problems.  Here is a partial list.
1. While configuring the machine I could not find  COM interface. I contacted HotTop believing that it is the roaster that offers the interface and I am waiting for their response.  The roaster offers UART.  Is this something new?  Is there additional piece necessary, like USB UART converter?  As is, I don't think UART allows to control the roaster. Did anybody else run into this problem?
2. I found the PID control window and set the parameters.  The string of actions that is necessary to bring it to forefront is a little unusual, as, for example, described in this video: https://www.youtube.com/watch?v=ykuUCXhGAC4.  Now, I can not re-open that window: the CONTROL button does not bring it on.  Is there an alternative way to open it?   
3. I was unsuccessful in finding the way to preheat the roaster to a certain initial temperature and keep it until I drop the beans.  I would greatly appreciate help with setting it up

**Comment by poundy** [comment]
1. COM is UART. Both are just different names for Serial Communication ports.

**Reply by romanplains** [reply]
Thank you !
